### ISSUE TITLE
[Bug] Sorting isn't applied when data is refreshed

### ISSUE THREAD

**Issue by mattpurlandncc** [vincjo/datatables#18]
Even though getSorted() is unchanged and returns the sorting object, the rows aren't sorted when the data is refresh

**Comment by mattpurlandncc** [comment]
@vincjo it may just be a result of my stack, I'm not sure. I'm using Svelte with Laravel and Inertia. But when the underlying data is changed I'm doing the following:

```
$: if(underlyingData) {
    // Underlying data has been refresh via an Inertia request via the page prop, update the table handler
    handler.setRows([...underlyingData]);
}

```
Both filters and search are reapplied as expected, but sorting isn't

**Comment by vincjo** [comment]
I reproduced the same bug with my stack.
I'll make a quick fix, thx for point it out

**Comment by mattpurlandncc** [comment]
No problem, glad I could help

**Comment by vincjo** [comment]
Problem solved with `1.4.0`
